src/main/java/com/internet/shop/controllers/cart/admin/DeleteProductFromCartAdminController.java

```diff
@@ -20,8 +19,11 @@
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        Long id = Long.valueOf(req.getParameter("id"));
-        cartService.deleteProduct(cartService.getByUserId(USER_ID), productService.get(id));
-        resp.sendRedirect(req.getContextPath() + "/carts/edit");
+        Long productId = Long.valueOf(req.getParameter("productId"));
+        Long cartId = Long.valueOf(req.getParameter("cartId"));
+        cartService.deleteProduct(cartService.get(cartId),
+                productService.get(productId));
+        req.setAttribute("cart", cartService.get(cartId));
```

Comment:
let's save cartService.get() to local variable

Reply:
Fixed to new variable userCart